internal/data/spicedb.go

```diff
@@ -70,6 +71,22 @@ func NewSpiceDbRepository(c *conf.Data, logger log.Logger) (*SpiceDbRepository,
 		return nil, nil, err
 	}
 
+	// wait upto 30 seconds to confirm spicedb is connected
+	ctx, cancel := context.WithTimeout(context.Background(), time.Second*30)
+	defer cancel()
+
+	_, err = client.ReadSchema(ctx, &v1.ReadSchemaRequest{}, grpc.WaitForReady(true))
+	if err != nil {
+		return nil, nil, fmt.Errorf("error trying to reach SpiceDB: %w", err)
```

Comment:
@wscalf and myself dicussed this a bit, it wouldnt be ideal to return a error here.

Reply:
After discussing with @wscalf, we agreed that due to https://github.com/project-kessel/relations-api/pull/111, that adding logging/erroring here at all is redundant and not very useful. This only tells us the connection state at one point in time, while the work done with the readyz endpoint allows to check the state at any point.

Reply:
I assume we can close this then @lennysgarage

Reply:
Yes, I'll go ahead with it